nestkernel/node_manager.cpp

```diff
@@ -186,20 +186,14 @@ NodeManager::add_node( index model_id, long n )
 void
 NodeManager::add_neurons_( Model& model, index min_node_id, index max_node_id, NodeCollectionPTR nc_ptr )
 {
-  // upper limit for number of neurons per thread; in practice, either
-  // max_new_per_thread-1 or max_new_per_thread nodes will be created
   const size_t num_vps = kernel().vp_manager.get_num_virtual_processes();
-  const size_t max_new_per_thread =
-    static_cast< size_t >( std::ceil( static_cast< double >( max_node_id - min_node_id + 1 ) / num_vps ) );
 
 #pragma omp parallel
   {
     const index t = kernel().vp_manager.get_thread_id();
 
     try
     {
-      model.reserve_additional( t, max_new_per_thread );
```

Comment:
Could it make sense to keep `reserve_additional` to avoid unnecessary automatic vector resizings?